src/main/java/thestonedturtle/mahoganyhomes/MahoganyHomesPlugin.java

```diff
@@ -157,6 +174,11 @@ public void shutDown()
 	@Subscribe
 	public void onConfigChanged(ConfigChanged c)
 	{
+		if (c.getGroup().equals(PLANK_SACK_GROUP_NAME) && c.getKey().equals(PLANK_COUNT_KEY))
```

Comment:
Only do this is they currently have an active contract and the plugin hasn't "timed out". Many people leave the plugin on at all times even if they aren't actively doing the content.